src/components/ProductCard/index.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import { Meta, StoryObj } from "@storybook/react";
+import {ProductCard} from ".";
+
+const meta: Meta<typeof ProductCard> = {
+  title: "Components/ProductCard",
+  component: ProductCard,
+  parameters: {
+    layout: "centered",
+  },
+  tags: ["autodocs"],
+};
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  args: {
+    productCode: "00123456",
+    productDetail: {
+      productDetails: "Navy Blue printed woven empire %S, has a V-neck, three-quarter sleeves, and flared hem",
+      sizeAndFit: "Semi-Fit\nThe model (height 5'8\") is wearing a size S",
+      materialAndCare: "Cotton\nMachine-wash"
+    },
+    productSpecifications: {
```

Comment:
create a data folder
and assign all these required objects in it
like productSpecifications, product details

So we can use them directly